tests/tracking/test_rest_tracking.py

```diff
@@ -386,7 +386,12 @@ def test_log_model(mlflow_client, backend_store_uri):
                 tag = run.data.tags["mlflow.log-model.history"]
                 models = json.loads(tag)
                 model.utc_time_created = models[i]["utc_time_created"]
-                assert models[i] == model.to_dict()
+
+                history_model_meta = models[i].copy()
+                history_model_meta.pop("model_uuid")
+                model_meta = model.to_dict().copy()
+                model_meta.pop(("model_uuid"))
+                assert history_model_meta == model_meta
```

Comment:
Why do we need to exclude `model_uuid`?

Reply:
Because when each time we log the model, we will generate a random model uuid, which is a different value each time.

Reply:
Could we add an equivalency test of
```python
original_model_uuid = history_model_meta.pop("model_uuid")
...
new_model_uuid = model_meta.pop(("model_uuid"))
assert original_model_uuid != new_model_uuid
```
Just to make sure that we're never accidentally reusing a uuid?